code/modules/paperwork/paper.dm

```diff
@@ -146,11 +146,13 @@
 		onclose(user, name)
 	return
 
-/obj/item/paper/attack(mob/living/carbon/M, mob/living/carbon/user)
+/obj/item/paper/attack(mob/living/carbon/human/M, mob/living/carbon/user)
+
 	if(user.zone_selected == "eyes")
-		user.visible_message(SPAN_NOTICE("You show the paper to [M]. "),
-			SPAN_NOTICE(" [user] holds up a paper and shows it to [M]. "))
-		examine(M)
+		if(!isyautja(M) && !isxeno(M))
```

Comment:
none of the rest of this proc does anything if you're not a human so just put if(!ishuman(M)) return at the top and remove this check and the one for lipstick wiping

Reply:
yautja are still human so they still get flashed with a non human check